pytorch/training/docker/1.4.0/py3/cu101/Dockerfile.gpu

```diff
@@ -123,9 +125,14 @@ RUN /opt/conda/bin/conda config --set ssl_verify False \
 # custom versions from an S3 bucket. This will need to be removed in the future
 RUN pip install \
     --no-cache-dir smdebug==${SMDEBUG_VERSION} \
-    "sagemaker<2" \
+    "sagemaker>=2,<3" \
+    sagemaker-training \
```

Comment:
Please install `sagemaker-pytorch-training`, and not `sagemaker-training`, and remove the sagemaker-pytorch-training installed at the end of the file.

Reply:
See above